geocoder/admin0/data/wikipedia_countries_native_names.csv

```diff
@@ -46,15 +46,15 @@ TLS,East Timor,Timor Lorosa'e
 BHS,Bahamas,The Bahamas
 CHN,China,Zhōngguó
 EGY,Egypt,Misr or Masr
-,Equatorial Guinea,Guinea Ecuatorial
+GNQ,Equatorial Guinea,Guinea Ecuatorial

```

Comment:
modifying this file looks like "cheating to me" but ok if you know what you're doing :)

I mean, this is a raw source file, right?